README.md

```diff
@@ -45,6 +45,17 @@ Limitations:
 
 - It is not currently possible to disable periodics per job
 
+# Konflux
+
+The architecture of the Konflux configuration architecture is summarized in this diagram:
```

Comment:
Could you add some more info about design decisions made, so we may iterate on in the future.